frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -208,9 +187,13 @@ export default {
     },
     selectedSentence() {
       return (
-        this.sentences[this.itemNumber] && this.sentences[this.itemNumber].text
+        this.sentences[this.predictionNumber] &&
+        this.sentences[this.predictionNumber].text
       );
     },
+    isPreannotated() {
+      return (this.predictionsLength && !this.annotations.length) || false;
+    },
```

Comment:
since the left part is always a boolean, the test 'right expression || left expression' can be replace by 'right expression'